pkg/apis/deployment/annotations.go

```diff
@@ -26,6 +26,7 @@ const (
 	ArangoDeploymentPodChangeArchAnnotation         = ArangoDeploymentAnnotationPrefix + "/arch"
 	ArangoDeploymentPodRotateAnnotation             = ArangoDeploymentAnnotationPrefix + "/rotate"
 	ArangoDeploymentPodReplaceAnnotation            = ArangoDeploymentAnnotationPrefix + "/replace"
+	ArangoDeploymentPodFastRestart                  = ArangoDeploymentAnnotationPrefix + "/fast_restart"
```

Comment:
can we add here some short description how it works?

Reply:
actually it is draft PR, but yes I think we should have one place where all annotations should be described.